Examples/README-PingFederate.md

```diff
@@ -0,0 +1,52 @@
+# Using AppAuth for iOS and macOS with PingFederate
+
+This example uses the *PingFederate OAuth2 Playground* sample application to quickly stand up a new PingFederate server as an OpenID Connect Provider.
+
+Developer licenses and the PingFederate software can be found at https://developer.pingidentity.com/get-started.
+
+
+
+## You will need
+
+* PingFederate server & license ([download developer software and licenses](https://developer.pingidentity.com/get-started))
+* PingFederate OAuth2 Playground (available from [product downloads](https://www.pingidentity.com/en/products/downloads/pingfederate.html))
+
+
+
+## PingFederate configuration
+
+* Install PingFederate and the OAuth2 Playground (see the readme in the OAuth2 Playground distribution)
+* Modify the OAuth client `Authorization Code Client` in the PingFederate console:
+  * Edit the `Redirect URIs` option to add the example redirect URI (e.g. com.example.appauth://cb)
+  * If using the loopback interface with the macOS example, add the following redirect URI: `http://127.0.0.1:*/`
+* Save your changes
+
+Single sign-on using the SFSafariViewController expects a persistent session cookie to be used. This is not the default configuration for PingFederate, however there are [simple instructions to switch from using session cookies to persistent cookies](https://docs.pingidentity.com/bundle/pf_sm_extendLifetimeOfPfCookie_pf83/page/concept/extendingLifetimeOfPfCookie.html).
+
+**Note**: Due to the Application Transport Security (ATS) features of iOS9+, your PingFederate server must have a valid SSL certificate.
```

Comment:
NB. It is possible to add a domain to a temporary whitelist too, this is what I did for localhost for the AppAuth samples so it can be tested against a dev server.  Of course the risk is people add this whitelist during development, and forget to remove…